VoxelSniperCore/src/main/java/com/github/kevindagame/brush/CopyPastaBrush.java

```diff
@@ -107,7 +107,7 @@ private void doPasta(final SnipeData v) {
     }
 
     @Override
-    protected final void arrow(final com.thevoxelbox.voxelsniper.snipe.SnipeData v) {
+    protected final void arrow(final com.github.kevindagame.snipe.SnipeData v) {
```

Comment:
Do you know why we use fully qualified name here? Happens in more places...

Reply:
dunno